app/services/base_service.rb

```diff
@@ -0,0 +1,6 @@
+class BaseService
+  def fetch_json(path, params)
+    response = conn.get(path, params)
+    JSON.parse(response.body, symbolize_names: true)
+  end
+end
```

Comment:
Created a `BaseService` to DRY up the code and because both services currently utilize `.fetch_json`